notebooks/Media-Pretrained/02_Training_Layer_Recap.ipynb

```diff
@@ -111,9 +87,1051 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Retrieve your automated deployment variables\n",
+    "# How to train your Use Case Optimized Recommenders and Solution Versions\n",
+    "\n",
+    "As mentioned at the top of this notebook, a dataset group, schemas, datasets, solutions, and campaigns have already been created for you. You can open another browser tab/window to view these resources in the Personalize AWS Console.\n",
+    "\n",
+    "Bellow we will walk you though the steps we used to create there resorces, since they are already created, we will only be retrieving the automated deployment variables, however you can also run this code to train reources if you have not run the automation.\n",
+    "\n",
+    "Please take into account that creating these resources in your account will incur a cost and that it will take upload and training time to to these steps trough this notebook (several hours)."
```

Comment:
Corrected some spelling and missing words:

Please take into account that creating these resources in your account will incur a cost and that it will take upload and training time to complete these steps through this notebook (several hours).

Reply:
Fixing!